src/scripts/main.js

```diff
@@ -21,7 +21,19 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key of Object.keys(data)) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    if (data[key].length !== 0) {
```

Comment:
The condition `data[key].length !== 0` is incorrect for checking if an object has properties. Objects do not have a 'length' property. Instead, use `Object.keys(data[key]).length !== 0` to check if the object has any properties.